ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -24,11 +24,14 @@ class SampleHandler: RPBroadcastSampleHandler {
     static let kBroadcastSetupInfoRoomNameKey = "roomName"
 
     // In order to save memory, we request that our source downscale its output.
-    static let kDownScaledMaxWidthOrHeight = 720
+    static let kDownScaledMaxWidthOrHeight = UInt(720)
 
     // Maximum bitrate (in kbps) used to send video.
     static let kMaxVideoBitrate = UInt(1400)
 
+    // Maximum frame rate to send video at.
+    static let kMaxVideoFrameRate = UInt(15)
```

Comment:
This is tricky. 15 make sense in most cases, however if you are watching a video and broadcasting, should it be 30?

Reply:
I will explore an option where kMaxVideoFrameRate is fixed at 15 until video content is detected. At this point, I will issue another format request to the Source to raise the limit to the video content's natural frame rate (20 - 25 fps). If the video content is no longer detected then return to 15 fps.

As for content detection, I am looking for output from ReplayKit that is at a constant frame rate between 24-25 fps, determined by a rolling average of frame rate.

Reply:
I've updated the PR to demonstrate pinning to `kMaxVideoFrameRate`, and boosting to match the natural rate of video content.